Project/Resources/Markdown.xshd

```diff
@@ -0,0 +1,73 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<SyntaxDefinition name="Markdown" extensions=".md">
+    <RuleSets>
+
+        <RuleSet name="HtmlTagSet" ignorecase="false">
+            <Delimiters>/=</Delimiters>
+
+            <Span name="String" stopateol="true" color="#0451A5" bold="false" italic="false">
+                <Begin>"</Begin>
+                <End>"</End>
+            </Span>
+
+            <Span name="Char" stopateol="true" color="Blue" bold="false" italic="false">
+                <Begin>'</Begin>
+                <End>'</End>
+            </Span>
+
+            <MarkPrevious color="Red" bold="false" italic="false">=</MarkPrevious>
+
+            <KeyWords name="Slash" color="DarkMagenta" bold="false" italic="false">
+                <Key word="/"/>
+            </KeyWords>
+
+            <KeyWords name="Assignment" color="Black" bold="false" italic="false">
+                <Key word="="/>
+            </KeyWords>
+        </RuleSet>
+
+        <RuleSet ignorecase="true">
+
+            <Span name="Link" stopateol="true" color="Maroon" italic="false">
+                <Begin>[</Begin>
+                <End>]</End>
+            </Span>
+
+            <Span name="LinkUrl" stopateol="true" color="Purple" italic="false">
+                <Begin>(http</Begin>
+                <End>)</End>
+            </Span>
+
+            <Span name="Header4" stopateol="true" bold="true" color="LightBlue">
+                    <Begin startofline="true">####</Begin>
+            </Span>
+            <Span name="Header3" stopateol="true" bold="true" color="DodgerBlue">
+                    <Begin startofline="true">###</Begin>
+            </Span>
+
+            <Span name="Header2" stopateol="true" bold="true" color="Blue">
+                    <Begin startofline="true">##</Begin>
+            </Span>
+
+            <Span name="Header2" stopateol="true" bold="true" color="DarkBlue">
+                    <Begin startofline="true">#</Begin>
+            </Span>
+
+            <Span name="Bold" color="Black" bold="true" stopateol="true">
+                <Begin>**</Begin>
+                <End>**</End>
+            </Span>
+
+            <Span name="Italic" color="Black" italic="true" stopateol="true">
+                <Begin>__</Begin>
+                <End>__</End>

```

Comment:
Single `_` gives italic too:

```
Apple _banana_ carrot
```

Apple _banana_ carrot

Reply:
done.

Reply:
GH shows two __ still

Reply:
@gerhardol it's comment was to add a new rule to **also** support single '_' that was added elsewhere, so not marking this comment as resolved...